ga4gh/datamodel/ontologies.py

```diff
@@ -26,11 +29,28 @@ def add(self, id_, name):
         self._nameIdMap[id_] = name
         self._idNameMap[name] = id_
 
+    def getLocalId(self):
+        return self._localId
+
     def getId(self, name):
```

Comment:
``getId`` is a bit confusing here, and would usually imply that we're returning the ID of the actual Ontology object itself, and not doing a search. ``getById`` and ``getByName`` perhaps? I realise these aren't part of this PR, so feel free to shelve this for later.